### PR TITLE
DevEx: More validations!

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: docs/entities/WorkItem.md [hunk 1]
```diff
@@ -131,6 +131,7 @@
       type: "string"
       flags: 
         presence: "required"
+        description: "Unique case ID in XXXXX-YY format."
```
Comment (rachelschneiderman) [inline docs/entities/WorkItem.md]: Should this say unique identifier? So as not to be confused with caseId?

File: shared/src/business/entities/cases/Case.js [hunk 2]
```diff
@@ -447,7 +447,7 @@ Case.VALIDATION_RULES = {
     .description('Case caption before modification.'),
   initialDocketNumberSuffix: joi
     .string()
-    .max(2) // TODO: add enumerator
+    // .valid(...Object.values(DOCKET_NUMBER_SUFFIXES))
```
Comment (rachelschneiderman) [inline shared/src/business/entities/cases/Case.js]: Is this commented out on purpose?

File: shared/src/business/entities/WorkItem.js [hunk 3]
```diff
@@ -101,7 +101,11 @@ joiValidationDecorator(
       .allow(null),
     completedMessage: joi.string().max(100).optional().allow(null),
     createdAt: joiStrictTimestamp.optional(),
-    docketNumber: joi.string().regex(DOCKET_NUMBER_MATCHER).required(),
+    docketNumber: joi
+      .string()
+      .regex(DOCKET_NUMBER_MATCHER)
+      .required()
+      .description('Unique case ID in XXXXX-YY format.'),
```
Comment (rachelschneiderman) [inline shared/src/business/entities/WorkItem.js]: Identifier?

**Review by rachelschneiderman — APPROVED** [review]

File: shared/src/business/entities/WorkItem.js (hunk 3 above)
Comment (ericsorenson) [reply shared/src/business/entities/WorkItem.js]: Good point!

File: docs/entities/WorkItem.md (hunk 1 above)
Comment (ericsorenson) [reply docs/entities/WorkItem.md]: Good point!

File: shared/src/business/entities/cases/Case.js (hunk 2 above)
Comment (ericsorenson) [reply shared/src/business/entities/cases/Case.js]: No, good point, I'll re-enable that.

**Pull request merged**